nautobot/cloud/tables.py

```diff
@@ -39,6 +40,53 @@ class Meta(BaseTable.Meta):
         )
 
 
+class CloudNetworkTable(BaseTable):
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    cloud_type = tables.Column(linkify=True)
+    cloud_account = tables.Column(linkify=True)
+    parent = tables.Column(linkify=True)
+    actions = ButtonsColumn(CloudNetwork)
+
+    class Meta(BaseTable.Meta):
+        model = CloudNetwork
+        fields = (
+            "pk",
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "prefixes",
+            "tags",
+            "actions",
+        )
+        default_columns = (
+            "pk",
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "actions",
+        )
+
+
+class CloudNetworkPrefixAssignmentTable(BaseTable):
+    cloud_network = tables.Column(
+        verbose_name="Cloud Network",
+        linkify=lambda record: record.cloud_network.get_absolute_url(),
+        accessor="cloud_network.name",
+    )
+    prefix = tables.Column(linkify=True)
+    rd = tables.Column(accessor="vrf.rd", verbose_name="RD")
+    tenant = TenantColumn(accessor="vrf.tenant")
```

Comment:
Are these columns necessary?

Reply:
These are the same columns that are on the other prefixes table, and seem to be good columns for prefixes.